app/models/card.rb

```diff
@@ -0,0 +1,7 @@
+class Card < ActiveRecord::Base
+  belongs_to :deck
+
+  validates :question, presence: true 
+  validates :answer, presence: true 
+  validates :deck_id, presence: true 
```

Comment:
do we need to validate the deck_id? i feel like active record does this itself..